### PR TITLE
Webpack and cleanup.

### PR DESCRIPTION
**Pull request by fjlaubscher**
Library output was broken and very bloated.
Seems to be working fine now.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
     "storybook": "start-storybook -p 9001 -c .storybook",
     "deploy-storybook": "storybook-to-ghpages",
     "lint": "eslint -c .eslintrc src",
     "build-storybook": "build-storybook",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "start": "npm run storybook",
+    "prebuild": "npm run clean",
+    "build": "webpack --mode production --config webpack/config.js",
+    "clean": "rimraf build"
```
Comment (seagyn) [inline package.json]: What does this and prebuild do? I've never had to use either.

File: src/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import Dropdown from './dropdown';
 import Header from './header';
 import Footer from './footer';
 
-export default {
+const Oxygen = {
```
Comment (seagyn) [inline src/index.js]: How do we consume this on the "client" side?

Comment (fjlaubscher) [reply src/index.js]: Exactly the same as you would, nothing changed other than it being a named export.

Comment (seagyn) [reply src/index.js]: Oh right!

File: package.json (hunk 1 above)
Comment (fjlaubscher) [reply package.json]: `rimraf` is a npm package which allows you to delete a directory on both unix & windows. `rm -rf` only works on unix for example.

The prebuild hook just fires the clean script (which is the rimraf thing) to just clean the directory.
It's probably not completely necessary to keep deleting and recreating the folder, but it just keeps things a lot cleaner, especially when working with different outputs / tweaking the config.

**Pull request merged**